webextension/css/popup.css

```diff
@@ -1,16 +1,55 @@
 /* General Rules and Resets */
-body {
-  inline-size: 300px;
-  max-inline-size: 300px;
+* {
+  font-size: inherit;
+  margin-block-end: 0;
+  margin-block-start: 0;
+  margin-inline-end: 0;
+  margin-inline-start: 0;
+  padding-block-end: 0;
+  padding-block-start: 0;
+  padding-inline-end: 0;
+  padding-inline-start: 0;
```

Comment:
Yeah it's to prevent standard CSS breaking things like checkboxes and other items. If things have not been specified they should be fixed elsewhere.